### PR TITLE
Rewrite the session server to support RPC

### PR DESCRIPTION
**Pull request by ababo**
Resolves BV-972

### CODE REVIEW

File: sess/handler_test.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
+// +build !nosesstest
```
Comment (furkhat) [inline sess/handler_test.go]: no need in build tag for *_test.go files.

File: sess/product_test.go [hunk 2]
```diff
@@ -0,0 +1,116 @@
+// +build !nosesstest
```
Comment (furkhat) [inline sess/product_test.go]: same here

File: sess/handler_test.go (hunk 1 above)
Comment (ababo) [reply sess/handler_test.go]: The purpose of this tag is to exclude specific tests from test launches. Not sure how this is still relevant (long time ago we agreed on that to be able to exclude `eth` tests).

**Comment by sofabeat** [comment]
Con you, please remove SOMC in "resolve conflicts". It is gone. No more external SOMC, only internal SOMC server.

**Comment by ababo** [comment]
@sofabeat conflicts resolved

**Pull request merged**